cmd/immuclient/immuc/setcommands.go

```diff
@@ -111,19 +100,66 @@ func (i *immuc) VerifiedSet(args []string) (string, error) {
 
 	time.Sleep(1 * time.Millisecond)
 
-	value2, err := ioutil.ReadAll(&buf)
+	vi, err := i.Execute(func(immuClient client.ImmuClient) (interface{}, error) {
+		return immuClient.VerifiedGet(ctx, key)
+	})
 	if err != nil {
 		return "", err
 	}
 
-	vi, err := i.Execute(func(immuClient client.ImmuClient) (interface{}, error) {
-		return immuClient.VerifiedGet(ctx, key)
+	return PrintKV(vi.(*schema.Entry), true, false), nil
+}
+
+func (i *immuc) Restore(args []string) (string, error) {
+	key, atRevision, err := i.parseKeyArg(args[0])
+	if err != nil {
+		return "", err
+	}
+
+	if atRevision == 0 {
```

Comment:
`get key@0`  should be equivalent to `get key`,  then atRevision == 0 the error could refer to can not restore from current value ?

Reply:
Addressed in 98ff687545172cdd06d632354c77c8700d022352